extensions/wikia/EditPageLayout/EditPageLayoutController.class.php

```diff
@@ -206,6 +208,12 @@ public function executeEditPage() {
 			? wfMessage( 'editpagelayout-notificationsLink-none' )->escaped()
 			: wfMessage( 'editpagelayout-notificationsLink', count( $this->notices ) )->parse();
 
+		$templateType = ( new TemplateClassificationService() )
+			->getType( $wgCityId, $this->title->getArticleID() );
```

Comment:
@RafLeszczynski Good point. @katanka My suggestion would be to catch exception and then not display the link but also make sure to log exception (look in other place how things are being logged).